### PR TITLE
Don’t generate BasicFormat+Extensions.swift and SyntaxClassifcation.swift

### PR DESCRIPTION
**Pull request by ahoppen**
Instead of generating these files, we can just define them directly.

Also: Remove a couple of unused properties in `TokenSpec`.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

File: Sources/SwiftBasicFormat/BasicFormat.swift [hunk 1]
```diff
@@ -164,7 +164,43 @@ open class BasicFormat: SyntaxRewriter {
 
   /// Whether a leading newline on `token` should be added.
   open func requiresIndent(_ node: some SyntaxProtocol) -> Bool {
-    return node.requiresIndent
```
Comment (bnbarham) [inline Sources/SwiftBasicFormat/BasicFormat.swift:167]: The nice thing about this being generated is that when a node is added, `isIndented` is right in the constructor and would *hopefully* have some thought put into it. Where as now you have to know to also update `BasicFormat`. Maybe that's fine (it's a similar case for the whitespace now), but seems worth mentioning.

**Review by bnbarham — APPROVED** [review]

File: Sources/SwiftBasicFormat/BasicFormat.swift (hunk 1 above)
Comment (ahoppen) [reply Sources/SwiftBasicFormat/BasicFormat.swift:167]: That’s a price I’m willing to pay for the reduced complexity. And I don’t really expect many new nodes that require indentation anyway 😉

**Pull request merged**